apps/account-api/src/main.ts

```diff
@@ -42,6 +42,18 @@ async function bootstrap() {
     type: VersioningType.URI,
   });
 
+  const swaggerDoc = await generateSwaggerDoc(app, {
+    title: 'Account Service',
+    description: 'Account Service API',
+    version: '1.0',
+  });
+
+  const args = process.argv.slice(2);
+  if (args.find((v) => v === '--writeOpenApi')) {
+    writeOpenApiFile(swaggerDoc, './openapi-specs/account.openapi.json');
+    process.exit(0);
+  }
+
```

Comment:
Moved here from `buld-openapi.ts`. Common to both runtime Swagger UI, and OpenAPI spec file generation.